Expensify/App
# Feature: Make lonely emojis bigger

### Details

Increases the fontSize of the `ReportActionItemFragment` of type `COMMENT` when it's composed of a single emoji. To do that, we: 
1. Add a new style: `singleEmojiText`, with custom fontSize and lineHeight
2. Add a new const: a regex expression to match emojis `CONST.REGEX.EMOJIS`
3. Create `isSingleEmoji`, a function that validates if a string is composed of a single emoji
4. In the `ReportActionItemFragment` we check if the `COMMENT` passes our `isSingleEmoji` validation. If it passes, we use the `singleEmojiText` style instead of our default.

### Fixed Issues

Fixes #3712 

### Tests

1. Open any conversation.
2. Send a message composed of a single emoji.
3. Send a second message composed of multiple emojis (or emoji + text)
3. The lonely emoji on the first message is bigger.
### QA Steps

1. Open any conversation.
2. Send a message composed of a single emoji.
3. Send a second message composed of multiple emojis (or emoji + text)
3. The lonely emoji on the first message should be bigger.
### Tested On


### Screenshots


#### Web

https://user-images.githubusercontent.com/53711423/123180220-3d485980-d461-11eb-99da-ac9554b056c3.mov

#### Mobile Web

https://user-images.githubusercontent.com/53711423/123185158-a634cf00-d46b-11eb-9d4b-8da36ac9ac82.mp4

#### Desktop

https://user-images.githubusercontent.com/53711423/123180672-19394800-d462-11eb-8a07-0f8610958488.mov

#### iOS

https://user-images.githubusercontent.com/53711423/123185354-1cd1cc80-d46c-11eb-8169-1b9a2c231368.mp4

#### Android

https://user-images.githubusercontent.com/53711423/123186929-8dc6b380-d46f-11eb-909a-e46c706d3f65.mp4
